### PR TITLE
chore: add stdout and stderr to integration logs

### PR DESCRIPTION
**Pull request by ybt195**
# Test Plan
- UI change:
  - [ ] add screenshots
  - [ ] React? build & check storybooks
- [ ] user-facing api change: modify documentation and examples
- [ ] user-facing api change: add the "User-facing API Change" label
- [ ] bug fix: add regression test
- [ ] bug fix: determine if there are other similar bugs in the codebase
- [ ] new feature: add test coverage for any user-facing aspects
- [ ] refactor: maintain existing code coverage

### CODE REVIEW

File: cli/determined_cli/experiment.py [hunk 1]
```diff
@@ -235,7 +235,7 @@ def _parse_config_file_or_exit(config_file: io.FileIO) -> Dict:
 
 @authentication_required
 def submit_experiment(args: Namespace) -> None:
-    experiment_config = _parse_config_file_or_exit(args.config_file)
+    experiment_config = _parse_config_file_or_exit(args.config_file + "Fake name")
```
Comment (brain-good) [inline cli/determined_cli/experiment.py:238]: remove?

File: tests/integrations/command/command.py [hunk 2]
```diff
@@ -95,6 +95,10 @@ def get_command(id: str) -> Dict[str, Any]:
 def get_command_config(command_type: str, id: str) -> str:
     assert command_type in ["command", "notebook", "shell"]
     command = ["det", "-m", conf.make_master_url(), command_type, "config", id]
-    completed_process = subprocess.run(command, universal_newlines=True, stdout=subprocess.PIPE)
-    assert completed_process.returncode == 0
+    completed_process = subprocess.run(
+        command, universal_newlines=True, capture_output=True, env={"DET_DEBUG": "true"}
```
Comment (brain-good) [inline tests/integrations/command/command.py:99]: `DET_DEBUG` is ridiculously verbose, I don't think we should be setting it here

File: cli/determined_cli/experiment.py (hunk 1 above)
Comment (ybt195) [reply cli/determined_cli/experiment.py:238]: Wanted to see it run in CI first, will remove after.

File: tests/integrations/command/command.py (hunk 2 above)
Comment (ybt195) [reply tests/integrations/command/command.py:99]: DET_DEBUG only tells the CLI to print out the full stack trace on a CLI failure. This does not effect the logging level.

**Pull request closed without merging**